packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -242,7 +242,7 @@ export const VoiceProvider: FC<VoiceProviderProps> = ({
           updateError({ type: 'socket_error', message });
         }
       },
-      [client.readyState],
```

Comment:
the `client` dep needs to be updated here too, or it will also trigger too many rerenders